src/brsTypes/components/RoDeviceInfo.ts

```diff
@@ -0,0 +1,759 @@
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("text/size"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("text/color"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("text/opacity"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("background/color"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("background/opacity"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("window/color"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("window/opacity"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("track"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("track_composite"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("track_analog"))) {
+            return new BrsString("default");
+        }
+        if (option.equalTo(new BrsString("muted"))) {
+            return new BrsString("default");
+        }
+
+        return new BrsString("");
+    }
+
+    getEnableAppFocusStatus() {
+        return this.enableAppFocus;
+    }
+
+    setEnableAppFocusStatus(enable: BrsBoolean) {
+        this.enableAppFocus = enable;
+    }
+
+    getEnableScreenSaverExitedStatus() {
+        return this.enableScreenSaverExited;
+    }
+
+    setEnableScreenSaverExitedStatus(enable: BrsBoolean) {
+        this.enableScreenSaverExited = enable;
+    }
+
+    getEnableLowGeneralMemoryStatus() {
+        return this.enableLowGeneralMemory;
+    }
+
+    setEnableLowGeneralMemoryStatus(enable: BrsBoolean) {
+        this.enableLowGeneralMemory = enable;
+    }
+
+    getEnableLinkStatus() {
+        return this.enableLinkStatus;
+    }
+
+    setEnableLinkStatus(enable: BrsBoolean) {
+        this.enableLinkStatus = enable;
+    }
+
+    getEnableAudioGuideChangedStatus() {
+        return this.enableAudioGuideChanged;
+    }
+
+    setEnableAudioGuideChangedStatus(enable: BrsBoolean) {
+        this.enableAudioGuideChanged = enable;
+    }
+
+    getEnableCodecCapChangedStatus() {
+        return this.enableCodecCapChanged;
+    }
+
+    setEnableCodecCapChangedStatus(enable: BrsBoolean) {
+        this.enableCodecCapChanged = enable;
+    }
+
+    setVideoFormat(videoFormat: ValueKind.Object) {
+        this.videoFormat = videoFormat;
+    }
+
+    getVideoFormat() {
+        return this.videoFormat;
+    }
+
+    setAudioFormat(audioFormat: ValueKind.Object) {
+        this.audioFormat = audioFormat;
+    }
+
+    getAudioFormat() {
+        return this.audioFormat;
+    }
```

Comment:
Since nothing outside of this component is using these getters and setters, consider removing them in favor of reading and writing the private variables `this.audioFormat` directly!  It's less code to maintain and a bit of complexity we can remove.